### PR TITLE
tests: Keep cache in cleanup and rename for consistency

### PR DESCRIPTION
**Pull request by spetrosi**

### CODE REVIEW

File: tests/playbooks/cleanup.yml [hunk 1]
```diff
@@ -44,10 +50,11 @@
 
     - name: Remove related files
       shell: >-
-        rm -rf /var/opt/mssql
+        rm -rfv /var/optq/mssl
```
Comment (richm) [inline tests/playbooks/cleanup.yml:53]: optq?

Comment (spetrosi) [reply tests/playbooks/cleanup.yml]: I'm just blind...

File: tests/playbooks/cleanup.yml [hunk 2]
```diff
@@ -44,10 +50,11 @@
 
     - name: Remove related files
       shell: >-
-        rm -rf /var/opt/mssql
+        rm -rfv /var/opt/mssl
```
Comment (richm) [inline tests/playbooks/cleanup.yml:53]:
```suggestion
        rm -rfv /var/opt/mssql
```
?

Comment (spetrosi) [reply tests/playbooks/cleanup.yml]: shoot I need a break

**Pull request merged**